.travis.yml

```diff
@@ -3,7 +3,7 @@ language: python
 services:
   - docker
 install:
-  - pip install molecule docker
+  - pip install molecule docker ansible-lint flake8
```

Comment:
this is related to suse or ubuntu support?

Reply:
Also - we are sticking with molecule v2 for now - so please change this to `molecule<3` - and revert the other changes below related to molecule v3 support